sync/cli/__init__.py

```diff
@@ -52,25 +52,35 @@ def configure():
         default=(CONFIG.default_prediction_preference or Preference.BALANCED).value,
     )
 
-    db_host = click.prompt(
-        "Databricks host (prefix with https://)",
-        default=DB_CONFIG.host if DB_CONFIG else OPTIONAL_DEFAULT,
-    )
-    db_token = click.prompt(
-        "Databricks token",
-        default=DB_CONFIG.token if DB_CONFIG else OPTIONAL_DEFAULT,
-        hide_input=True,
-        show_default=False,
-    )
+    dbx_host = OPTIONAL_DEFAULT
+    dbx_token = OPTIONAL_DEFAULT
+    dbx_region = OPTIONAL_DEFAULT
+    if click.confirm("Would you like to configure a Databricks workspace?"):
+        dbx_host = click.prompt(
+            "Databricks host (prefix with https://)",
+            default=DB_CONFIG.host if DB_CONFIG else OPTIONAL_DEFAULT,
+        )
+        dbx_token = click.prompt(
+            "Databricks token",
+            default=DB_CONFIG.token if DB_CONFIG else OPTIONAL_DEFAULT,
+            hide_input=True,
+            show_default=False,
+        )
+        dbx_region = click.prompt(
+            "Databricks AWS Region name",
+            default=DB_CONFIG.aws_region_name if DB_CONFIG else OPTIONAL_DEFAULT,
+        )
```

Comment:
Instead of requiring users to specify a `region_name` as an argument to the various Databricks functions, I thought it would be a bit nicer if they could just configure that alongside their DBX host/token. Since a Databricks workspace can only exist in one region, this value is pretty static.